src/v1.0.0/2. Getting Started/0. Using the Wallet SDK.md

```diff
@@ -29,10 +31,63 @@ npm install --save @fuel-js/wallet
 </script>
 ```
 
+Creating a Wallet
+---
+The Fuel wallet can be used with or without a standard `web3` or `ethers` provider. The provider will be used for account signing or any Ethereum related actions, such as deposits and withdrawals.
+
+If an override `privateKey` is not specified, the first `account` given by the provider will be used.
+
+If an Ethereum `network` is not specified (e.g. `rinkeby`), it will be requested from the `provider`.
+
+```js
+import fuel from '@fueljs/wallet';
+
+// no provider, privateKey is generated
+const wallet = fuel.wallet(null);
+
+// web3 provider, first account used, network gathered from provider
+const wallet = fuel.wallet(window.web3.currentProvider);
+
+// provider with override privateKey and network specified
+const wallet = fuel.wallet(window.web3.currentProvider, {
+  privateKey: fuel.utils.hexlify(fuel.utils.randomBytes(32)),
+  network: 'rinkeby',
+});
+```
+
+Fauceting
+---
+
+All Fuel test networks have a token Faucet that will disperse fake tokens to requesting accounts (within several seconds to minutes).
+
+The faucet will disperse `1000` `faucetTokens`.
+
+The faucet test token identifier will always be available in `fuel.contants.faucetToken`.
+
+```js
+import fuel from '@fueljs/wallet';
+
+// new wallet
+const wallet = fuel.wallet(null, {
+  network: 'rinkeby',
+});
+
+// faucet fake tokens
+await wallet.faucet();
+
+// check your balance
+console.log('my balance: ', await wallet.balance(fuel.constants.faucetToken));
+```
+
+
 Depositing
 ---
 
-Let's try depositing `1` unit of the `DAI` ERC-20 token into Fuel. This assumes the current wallet has this amount of `DAI` on Ethereum to deposit.
+You can deposit ERC20 tokens or Ether into Fuel by using the `deposit` method. Simply specify the `token`, and `amount` to deposit.
+
+Because Fuel uses Ethereum key pairs, this process can be done directly with your provided MetaMask account, for example.
```

Comment:
```suggestion
You can deposit ERC-20 tokens or Ether into Fuel by using the `deposit` method. Simply specify the `token` and `amount` to deposit. Since Fuel uses Ethereum key pairs, this process can be done directly with your provided MetaMask account, for example.
```